### ISSUE TITLE
UAT: Content mismatch in UAT & PROD environment

### ISSUE THREAD

**Issue by mayank2k2** [OUPMATH/OxfrodMaths_Prod#27]
as discussed in today' call  i m creating new issue that will sync  content issue between QC, UAT & PROD environments.

**Comment by manasghosh81** [comment]
Please check this issue.
all the environments are updated with content files and db table has  been  updated.

The db table content 805 rows and total 13182 files for content.If any content is not working in any environment  please let me know.

**Comment by mayank2k2** [comment]
Homework not working on following items

![image](https://cloud.githubusercontent.com/assets/16239450/15171524/d0bedc96-176c-11e6-8390-36ce02372c28.png)

**Comment by manasghosh81** [comment]
@mayank2k2 :Could you please let me know which class these content belongs to

**Comment by mayank2k2** [comment]
all the classes

**Comment by manasghosh81** [comment]
@Mayank,My question  the particular contents you mention belongs to which class because the screenshot does not contain the class code(3.4,5),so I can debug it

**Comment by mayank2k2** [comment]
@manasghosh81 these are from  class code(3.4,5) but you need to check all classes

**Comment by manasghosh81** [comment]
All the contents are loading .It takes more time due to mathematical expression.Please check with editorial team and let content development team to fix this issue

**Comment by mayank2k2** [comment]
@manasghosh81 , what do mean by saying "Please check with editorial team and let content development team to fix this issue"

These are opening in prod but not in UAT.

**Comment by mayank2k2** [comment]
@pankaj44 , you told yesterday that UAT content was replaced wth prod then why the issue is still not fixed ? have you checked.

**Comment by pankaj44** [comment]
Hi Mayank,
Could you please let me know which content is not working. I mean the content from UAT and production are synch.

**Comment by mayank2k2** [comment]
@pankaj44  same list has already included
![image](https://cloud.githubusercontent.com/assets/16239450/16110686/9c6ca578-33cc-11e6-997e-b78065d94761.png)

**Comment by pankaj44** [comment]
Hi Mayank,
I have checked all the content UAT and production both are synch. But few of the content are not loading because it is taking more time to load the content, in that case it seems like it is content issue. 
I have investigated the issue and found that it is related to Mathjax server. Ex.
I have open one content in UAT and it is taking approximate 33 minutes to load the content but in production it is working fine. The screen-shot is attached below

![image](https://cloud.githubusercontent.com/assets/16384541/16222456/bec41b3c-37b5-11e6-9ad0-9e2ddc54abae.png)
